### ISSUE TITLE
NTR: nasopharynx

### ISSUE THREAD

**Issue by cmrn-rhi** [cidgoh/GRDI_AMR_One_Health#1]
# AMR-GRDI Specification - New Term Request (NTR)

### **Associated Field(s):**  anatomical_part

**Label:** Nasopharynx
**Definition/Description:** The section of the pharynx that lies above the soft palate.
**Definition Source:** http://www.informatics.jax.org/accession/cwg
**Synonyms or alternative labels:** NP
**Additional Comments (not an annotation)**: 
_description of usage within a sample plan_

--- 
Contributor Attribution: https://orcid.org/0000-0002-1825-0097

**Comment by cmrn-rhi** [comment]
This is a pretend/example new term request.
